react-native/training/src/app/screens/BookDetails/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
```

Comment:
`BookDetails` is a child screen of `Dummy` screen. Move it to `src/app/screens/Dummy/screens/BookDetails`